roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -53,6 +54,10 @@
   when:
   - openshift_logging_image_pull_secret == ''
 
+- command: >
+    {{ openshift.common.client_binary }} annotate serviceaccount aggregated-logging-elasticsearch --overwrite
+    serviceaccounts.openshift.io/oauth-redirectreference.primary='{"kind":"OAuthRedirectReference","apiVersion":"v1","reference":{"kind":"Route","name":"proxy"}}'
```

Comment:
naming nit: if we ever use the proxy for another component in this space we'll have wished we called this something like `elasticsearch-proxy`